frontend/src/components/common/authorized-menu/styles.module.scss

```diff
@@ -6,3 +6,7 @@
   padding: 40px;
   background-color: var(--background-gray-300);
 }
+
+.bottomLine:not(:last-child) {
```

Comment:
Won't it be better to choose nav groups from menu class childs? then you won't need to pass conditional className prop,, or at least name this class more clear

Reply:
It can be done using smth like .menu > div:not(:last-child), but applying global styles is a bad practice